<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Michael Jansen</author>
 <class>SelectSchemeDialog</class>
 <widget class="KDialog" name="SelectSchemeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Shortcut Scheme</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>12</y>
     <width>671</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="whatsThis">
       <string>Select one of the standard KDE shortcut schemes</string>
      </property>
      <property name="text">
       <string>&amp;Standard scheme:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>m_schemes</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="KComboBox" name="m_schemes">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>1</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="whatsThis">
       <string>Select a shortcut scheme file</string>
      </property>
      <property name="text">
       <string>&amp;Path:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="buddy">
       <cstring>m_url</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="KUrlRequester" name="m_url">
      <property name="sizeIncrement">
       <size>
        <width>3</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="filter">
       <string notr="true">*.kksrc</string>
      </property>
      <property name="clickMessage">
       <string>Select Shortcut Scheme</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KDialog</class>
   <extends>QDialog</extends>
   <header>kdialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>kurlrequester.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
